opentelemetry-etw-logs/src/logs/converters.rs

```diff
@@ -9,33 +9,55 @@ pub(super) trait IntoJson {
 
 impl IntoJson for AnyValue {
     fn as_json_value(&self) -> Value {
-        match &self {
-            AnyValue::Int(value) => json!(value),
-            AnyValue::Double(value) => json!(value),
-            AnyValue::String(value) => json!(value.to_string()),
-            AnyValue::Boolean(value) => json!(value),
-            AnyValue::Bytes(_value) => todo!("No support for AnyValue::Bytes yet."),
-            AnyValue::ListAny(value) => value.as_json_value(),
-            AnyValue::Map(value) => value.as_json_value(),
-            &_ => Value::Null,
-        }
+        serialize_anyvalue(self, 0)
     }
 }
 
-impl IntoJson for HashMap<Key, AnyValue> {
-    fn as_json_value(&self) -> Value {
-        Value::Object(
-            self.iter()
-                .map(|(k, v)| (k.to_string(), v.as_json_value()))
-                .collect::<Map<String, Value>>(),
-        )
+const ERROR_MSG: &str = "Message truncated as nested lists/maps are not supported.";
```

Comment:
```suggestion
const ERROR_MSG: &str = "Value truncated as nested lists/maps are not supported.";
```


"Message" feels a bit specific as truncation could be in Body/Message or Attributes. is "Value" better? 
Or just "truncated as nested ..."

Reply:
Update the message to use "Value".